source/configuration_manual/fts/flatcurve.rst

```diff
@@ -0,0 +1,225 @@
+.. _fts_backend_flatcurve:
+
+Flatcurve FTS Engine
+====================
+
+.. versionadded:: 2.4.0;3.0.0
+
+Requires Xapian 1.4+ `<https://xapian.org/download/>`_ to work properly.
+
+This is a Dovecot FTS plugin to enable message indexing using the
+[Xapian](https://xapian.org/) Open Source Search Engine Library.
+
+The plugin relies on Dovecot to do the necessary stemming. It is intended
+to act as a simple interface to the Xapian storage/search query
+functionality.
+
+This driver supports match scoring and substring matches (on by default),
+which means it is RFC 3501 (IMAP4rev1) compliant. This driver does not
+support fuzzy searches.
+
+The driver passes all of the ImapTest search tests.
+`<https://imapwiki.org/ImapTest>`_
+
+Enabling flatcurve is designed to be as easy as adding these lines:
+
+.. code-block:: none
+
+    mail_plugins = $mail_plugins fts fts_flatcurve
+    plugin {
+        fts = flatcurve
+    }
+
+Optional parameters
+-------------------
+
+.. note:: The default settings should be fine in most scenarios.
+
+.. dovecot_plugin:setting:: fts_flatcurve_commit_limit
+  :plugin: fts-flatcurve
+  :default: 500
+  :values: @uint, !set to 0 to use the Xapian default
```

Comment:
Does the `!` do something useful here and below? Documentation says it makes the text reST, but the texts here look plaintext.

Also.. Looking at how this is used elsewhere, nowhere else does `:values: @something` follow with any text. So this likely should be added to the explanation text below.

Reply:
it prevents the following text from being highlighted like in `` quotes.
Putting the max/min/meaning on separate lines makes more dispersive to find quickly what you need to read, that is why I formatted it that way. I can move them to the text below, but I personally find that less effective when accessing the document.

There are other cases of this usage, for instance ``mail_crypt_require_encrypted_user_key`` and ``login_trusted_networks``